test/TestComponent.tsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
+import React from 'preact/compat';
 
-const TestComponent = (): JSX.Element => {
+const TestComponent = (): preact.JSX.Element => {
```

Comment:
I did not now how to fix this without prefixing the `JSX.Element` statement with `preact.`. It should be possible to do without, according to https://github.com/preactjs/preact/issues/1036